### PR TITLE
docs/protocol: Remove duplicate calibration request

### PR DESCRIPTION
**Pull request by Filiprogrammer**
Remove the compass calibration setting from the settings bit field,
as the compass can already be calibrated in another way.

### CODE REVIEW

File: docs/wifi_ble_communication_protocol.md [hunk 1]
```diff
@@ -274,9 +274,9 @@ The 2nd byte changes certain logging options.
 
 ### 3.1.1. Bit field – 1. Byte (Settings)
 
-| Bit 7 (MSB)         | Bit 6  | Bit 5       | Bit 4               | Bit 3           | Bit 2        | Bit 1    | Bit 0 (LSB) |
-|---------------------|--------|-------------|---------------------|-----------------|--------------|----------|-------------|
-| Compass calibration | Invite | Positioning | Collision Avoidance | Navigation-Mode | Explore-Mode | not used | not used    |
+| Bit 7 (MSB) | Bit 6  | Bit 5       | Bit 4               | Bit 3           | Bit 2        | Bit 1    | Bit 0 (LSB) |
```
Comment (LecraM2003) [inline docs/wifi_ble_communication_protocol.md:277]:
```suggestion
| Bit 7 (MSB) | Bit 6       | Bit 5               | Bit 4           | Bit 3        | Bit 2    | Bit 1    | Bit 0 (LSB) |
```

File: docs/wifi_ble_communication_protocol.md [hunk 2]
```diff
@@ -274,9 +274,9 @@ The 2nd byte changes certain logging options.
...
+|-------------|--------|-------------|---------------------|-----------------|--------------|----------|-------------|
```
Comment (LecraM2003) [inline docs/wifi_ble_communication_protocol.md:278]:
```suggestion
|-------------|-------------|---------------------|-----------------|--------------|----------|----------|-------------|
```

File: docs/wifi_ble_communication_protocol.md [hunk 3]
```diff
@@ -274,9 +274,9 @@ The 2nd byte changes certain logging options.
...
+| not used    | Invite | Positioning | Collision Avoidance | Navigation-Mode | Explore-Mode | not used | not used    |
```
Comment (LecraM2003) [inline docs/wifi_ble_communication_protocol.md:279]:
```suggestion
| Invite      | Positioning | Collision Avoidance | Navigation-Mode | Explore-Mode | not used | not used | not used    |
```

**Review by LecraM2003 — CHANGES_REQUESTED** [review]
Let the "not used" bit at bit 3, so it is consistant

**Comment by LecraM2003** [comment]
Also create an issue, where the protocol changes are adapted in client library and Alphabot.NET

**Comment by Filiprogrammer** [comment]
#duration: 1

**Review by LecraM2003 — APPROVED** [review]

**Pull request merged**